httpclient-parent/httpclient/src/main/java/io/yupiik/uship/httpclient/core/listener/impl/HARDumperListener.java

```diff
@@ -61,12 +62,33 @@ public class HARDumperListener implements RequestListener<HARDumperListener.Data
     private final Logger logger;
     private final Har har = new Har();
 
+    private boolean withStartedDateTime = true;
+    private boolean withTime = true;
+
     public HARDumperListener(final Path output, final Clock clock, final Logger logger) {
         this.output = output;
         this.clock = clock;
         this.logger = logger;
     }
 
+    public boolean isWithStartedDateTime() {
+        return withStartedDateTime;
+    }
+
+    public HARDumperListener setWithStartedDateTime(boolean withStartedDateTime) {
+        this.withStartedDateTime = withStartedDateTime;
+        return this;
+    }
+
+    public boolean isWithTime() {
+        return withTime;
+    }
+
+    public HARDumperListener setWithTime(boolean withTime) {
```

Comment:
SetWith is weirdish no? What about setEnableTime?

Reply:
Yes renaming it.